### PR TITLE
Added localization support

### PR DESCRIPTION
**Pull request by tutte**
Added localization support for spanish and engish languages.

### CODE REVIEW

**Comment by joemerlino** [comment]
@tutte did you fix this [issue](https://github.com/ZaidElkurdi/AssistantPlus/issues/1)? thanks in advance!

**Comment by tutte** [comment]
I'm afraid i did not, I fixed the kSecAttrAccessibleWhenUnlocked error and tapping in a search result is perfectly working for me (iOS 8.4, iPhone 7,2). I also updated the project to 8.4 sdk. I would suggest you to compile and try!

File: spotifyPlugin/spotifyPlugin/Localization.m [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+@interface Localization ()
+@end
+
+@implementation Localization
++(NSString *)localized:(NSString *)localization_key {
+    NSString* bundleString = [self.localization_bundle localizedStringForKey:localization_key value:@"" table:nil];
+    if (bundleString) {
+        return bundleString;
+    }
+    return localization_key;
```
Comment (ZaidElkurdi) [inline spotifyPlugin/spotifyPlugin/Localization.m]: Why return the key? I think it might be safer to always return bundleString instead of hiding a bug.

File: spotifyPlugin/spotifyPlugin/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+  Created by Matias on 19/8/15.
+  Copyright (c) 2015 Zaid Elkurdi. All rights reserved.
+*/
+"CATCH_PLAY" = "play";
+"CATCH_ON_SPOTIFY" = "spotify";
+"REGEX_PLAY_BY" = "(?:.*)Play (.*) by (.*) on Spotify";
+"REGEX_PLAY_SONG" = "(?:.*)Play (.*)on Spotify";
+"SEARCHING" = "Searching...";
+"CATCH_SEARCH" = "busca";
```
Comment (ZaidElkurdi) [inline spotifyPlugin/spotifyPlugin/Resources/en.lproj/Localizable.strings]: Still in Spanish

File: spotifyPlugin/spotifyPlugin/SpotifyPlayCommands.m [hunk 3]
```diff
@@ -112,4 +112,13 @@ - (NSString*)getSongForArtist:(NSString*)artistName fromSongs:(NSArray*)songs {
 }
 
 
+-(NSString*)localizedStringForKey:(NSString*)string_key {
```
Comment (ZaidElkurdi) [inline spotifyPlugin/spotifyPlugin/SpotifyPlayCommands.m]: Why is this here as well as Localization.h?

File: spotifyPlugin/spotifyPlugin/SpotifySearchCommands.m [hunk 4]
```diff
@@ -13,13 +13,13 @@ @implementation SpotifySearchCommands
 
 - (BOOL)handleSpeech:(NSString *)text withTokens:(NSSet *)tokens withSession:(id<APSiriSession>)session {
   //Do a quick/rough check first to see if we should do any more thorough processing
-  if ([tokens containsObject:@"search"] && [tokens containsObject:@"on"] && [tokens containsObject:@"spotify"]) {
+  if ([Localization localized:@"CATCH_SEARCH"] && [Localization localized:@"CATCH_ON_SPOTIFY"]) {
```
Comment (ZaidElkurdi) [inline spotifyPlugin/spotifyPlugin/SpotifySearchCommands.m]: Should be:
if ([tokens containsObject:[Localization localized:@"CATCH_SEARCH"]] && [tokens containsObject:[Localization localized:@"CATCH_ON_SPOTIFY"]]) {
